lib/go/test/flow_epoch_test.go

```diff
@@ -148,9 +148,44 @@ func TestEpochPhaseMetadataChange(t *testing.T) {
 		rewardIncreaseFactor)
 
 	t.Run("Should be able to change the configurable metadata during the staking auction", func(t *testing.T) {
+		// Should fail to set epoch config with invalid config sum of the staking phase and dkg phases is greater than epoch
+		tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateUpdateEpochConfigScript(env), idTableAddress)
+		_ = tx.AddArgument(cadence.NewUInt64(5))
+		_ = tx.AddArgument(cadence.NewUInt64(5))
+		_ = tx.AddArgument(cadence.NewUInt64(5))
```

Comment:
```suggestion
		_ = tx.AddArgument(cadence.NewUInt64(5)) // dkg
		_ = tx.AddArgument(cadence.NewUInt64(5)) // staking
		_ = tx.AddArgument(cadence.NewUInt64(5)) // epoch
```

Since the arguments are unnamed, some short comments can help readability.

Reply:
https://github.com/onflow/flow-core-contracts/pull/282/commits/c58ed63eddd13fdacde845bdf7120038bff021ad